### PR TITLE
chore: config review

### PR DESCRIPTION
**Pull request by sr-remsha**

### CODE REVIEW

**Comment by ai-dial-actions** [comment]
Hey there and thank you for opening this pull request! 👋🏼

We require pull request titles to follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) and it looks like your proposed title needs to be adjusted.

Details:

```
No release type found in pull request title "config review". Add a prefix to indicate what kind of release this pull request corresponds to. For reference, see https://www.conventionalcommits.org/

Available types:
 - fix
 - hotfix
 - feat
 - feature
 - chore
```

File: README.md [hunk 1]
```diff
@@ -33,44 +33,44 @@ Static settings are used on startup and cannot be changed while application is r
...
-| resources.cacheExpiration                  | 300000             | Expiration in milliseconds for synced resources in Redis
-| resources.compressionMinSize               | 256                | Compress a resource with gzip if its size in bytes more or equal to this value
-| redis.singleServerConfig.address           | -                  | Redis single server addresses, e.g. "redis://host:port"
-| redis.clusterServersConfig.nodeAddresses   | -                  | Json array with Redis cluster server addresses, e.g. ["redis://host1:port1","redis://host2:port2"]
-| invitations.ttlInSeconds                   | 259200             | Invitation time to live in seconds
-| access.admin.rules                         | -                  | Matches claims from identity providers with the rules to figure out whether a user is allowed to perform admin actions, like deleting any resource or approving a publication. Example: [{"source": "roles", "function": "EQUAL", "targets": ["admin"]}]. If roles contain "admin, the actions are allowed.
+| Setting                                    | Default            |Required| Description
```
Comment (sr-remsha) [inline README.md]: Please review the required column

File: README.md [hunk 2]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+|--------------------------------------------|--------------------|--------|-----------------------------------------------------------------------------------------------------------
+| config.files                               | aidial.config.json | No|Config files with parts of the whole config.
```
Comment (sr-remsha) [inline README.md]: add link to the example

Comment (sr-remsha) [reply README.md]: further in the documents it says that via the config.json you can supply the dynamic settings. Need to unify the description

File: README.md [hunk 3]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| config.reload                              | 60000              | No|Config reload interval in milliseconds.
```
Comment (sr-remsha) [inline README.md]: Add to the example file

File: README.md [hunk 4]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders                          | -                  | No|Map of identity providers. **Note**. At least one identity provider must be provided.
```
Comment (sr-remsha) [inline README.md]: Add all supported IDPs to the example.

Comment (sr-remsha) [reply README.md]: If we say that at least one provider must be provided - then is it a REQUIRED field?

File: README.md [hunk 5]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders.*.jwksUrl                | -                  | Optional|Url to jwks provider. **Required** if `disabledVerifyJwt` is set to `false`. *Note*. Either `jwksUrl` or `userInfoEndpoint` must be provided.
+| identityProviders.*.userInfoEndpoint       | -                  | Optional|Url to user info endpoint. *Note*. Either `jwksUrl` or `userInfoEndpoint` must be provided.
```
Comment (sr-remsha) [inline README.md]: Need example in the sample config

File: README.md [hunk 6]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+|--------------------------------------------|--------------------|--------|-----------------------------------------------------------------------------------------------------------
+| config.files                               | aidial.config.json | No|Config files with parts of the whole config.
+| config.reload                              | 60000              | No|Config reload interval in milliseconds.
+| identityProviders                          | -                  | No|Map of identity providers. **Note**. At least one identity provider must be provided.
+| identityProviders.*.jwksUrl                | -                  | Optional|Url to jwks provider. **Required** if `disabledVerifyJwt` is set to `false`. *Note*. Either `jwksUrl` or `userInfoEndpoint` must be provided.
+| identityProviders.*.userInfoEndpoint       | -                  | Optional|Url to user info endpoint. *Note*. Either `jwksUrl` or `userInfoEndpoint` must be provided.  
+| identityProviders.*.rolePath               | -                  | Yes|Path to the claim user roles in JWT token or user info response, e.g. `resource_access.chatbot-ui.roles` or just `roles`.
```
Comment (sr-remsha) [inline README.md]: Consider adding a reference to the external document https://learn.microsoft.com/en-us/entra/identity-platform/access-token-claims-reference
and supply more details on where and how to get the mentioned roles

File: README.md [hunk 7]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders.*.loggingKey             | -                  | No|User information to search in claims of JWT token.
```
Comment (sr-remsha) [inline README.md]: Add example in the sample config

Comment (sr-remsha) [reply README.md]: Supply a reference to the external documentation where users can get additional information

File: README.md [hunk 8]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders.*.loggingSalt            | -                  | No|Salt to hash user information for logging.
```
Comment (sr-remsha) [inline README.md]: Add example in the sample config

Comment (sr-remsha) [reply README.md]: Supply a reference to the external documentation where users can get additional information

File: README.md [hunk 9]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders.*.cacheSize              | 10                 | No|How many JWT tokens to cache.
+| identityProviders.*.cacheExpiration        | 10                 | No| How long to retain JWT token in cache.
+| identityProviders.*.cacheExpirationUnit    | MINUTES            | No|Unit of cache expiration.
```
Comment (sr-remsha) [inline README.md]: Other options than MINUTES?

File: README.md (hunk 4 above)
Comment (sr-remsha) [reply README.md]: Provide at least one full example with all available parameters

File: README.md [hunk 10]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders.*.issuerPattern          | -                  | No|Regexp to match the claim "iss" to identity provider
+| identityProviders.*.disableJwtVerification | false              | No|The flag disables JWT verification. *Note*. `userInfoEndpoint` must be unset if the flag is set to `true`.
```
Comment (sr-remsha) [inline README.md]: Add corresponding note to userInfoEndpoint parameter description

File: README.md [hunk 11]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders.*.userInfoEndpoint       | -                  | Optional|Url to user info endpoint. *Note*. Either `jwksUrl` or `userInfoEndpoint` must be provided.  
+| identityProviders.*.rolePath               | -                  | Yes|Path to the claim user roles in JWT token or user info response, e.g. `resource_access.chatbot-ui.roles` or just `roles`.
+| identityProviders.*.loggingKey             | -                  | No|User information to search in claims of JWT token.
+| identityProviders.*.loggingSalt            | -                  | No|Salt to hash user information for logging.
+| identityProviders.*.cacheSize              | 10                 | No|How many JWT tokens to cache.
+| identityProviders.*.cacheExpiration        | 10                 | No| How long to retain JWT token in cache.
+| identityProviders.*.cacheExpirationUnit    | MINUTES            | No|Unit of cache expiration.
+| identityProviders.*.issuerPattern          | -                  | No|Regexp to match the claim "iss" to identity provider
```
Comment (sr-remsha) [inline README.md]: Add example

File: README.md [hunk 12]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| identityProviders.*.disableJwtVerification | false              | No|The flag disables JWT verification. *Note*. `userInfoEndpoint` must be unset if the flag is set to `true`.
+| vertx.*                                    | -                  | No|Vertx settings.
```
Comment (sr-remsha) [inline README.md]: Add example

File: README.md [hunk 13]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| server.*                                   | -                  | No|Vertx HTTP server settings for incoming requests.
+| client.*                                   | -                  | No|Vertx HTTP client settings for outbound requests.
+| storage.provider                           | -                  | No|Specifies blob storage provider. Supported providers: s3, aws-s3, azureblob, google-cloud-storage, filesystem
```
Comment (sr-remsha) [inline README.md]: Add example for each supported storage provide. 
Add more info for filesystem. Can it really be configured?
Where can I get the full list of supported providers?

Comment (sr-remsha) [reply README.md]: Is it not required?

File: README.md [hunk 14]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| storage.endpoint                           | -                  | No|Optional. Specifies endpoint url for s3 compatible storages
```
Comment (sr-remsha) [inline README.md]: Optional = not required or there is an option?

Comment (sr-remsha) [reply README.md]: s3 compatible storages - why does it have to be S3 compatible?

File: README.md [hunk 15]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| storage.identity                           | -                  | Optional|Blob storage access key. Can be optional for filesystem, aws-s3, google-cloud-storage providers
```
Comment (sr-remsha) [inline README.md]: Better describe access-key value and other options

Comment (sr-remsha) [reply README.md]: Where do I get them?

File: README.md [hunk 16]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| storage.credential                         | -                  | Optional|Blob storage secret key. Can be optional for filesystem, aws-s3, google-cloud-storage providers
+| storage.bucket                             | -                  | No|Blob storage bucket
+| storage.overrides.*                        | -                  | No|Key-value pairs to override storage settings
```
Comment (sr-remsha) [inline README.md]: What does asterisk mean? 
Provide an example.
Provide a more clear description.

File: README.md [hunk 17]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| storage.createBucket                       | false              | No|Indicates whether bucket should be created on start-up
+| storage.prefix                             | -                  | No|Base prefix for all stored resources. Must not contain path separators or any invalid chars
```
Comment (sr-remsha) [inline README.md]: Provide example and the purpose of this parameter.

File: README.md [hunk 18]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| encryption.password                        | -                  | No|Password used for AES encryption
+| encryption.salt                            | -                  | No|Salt used for AES encryption
```
Comment (sr-remsha) [inline README.md]: Provide an example of a referent for an additional information

File: README.md [hunk 19]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| resources.maxSize                          | 1048576            | No|Max allowed size in bytes for a resource
+| resources.syncPeriod                       | 60000              | No|Period in milliseconds, how frequently check for resources to sync
+| resources.syncDelay                        | 120000             | No|Delay in milliseconds for a resource to be written back in object storage after last modification
+| resources.syncBatch                        | 4096               | No|How many resources to sync in one go
+| resources.cacheExpiration                  | 300000             | No|Expiration in milliseconds for synced resources in Redis
+| resources.compressionMinSize               | 256                | No|Compress a resource with gzip if its size in bytes more or equal to this value
+| redis.singleServerConfig.address           | -                  | No|Redis single server addresses, e.g. "redis://host:port"
```
Comment (sr-remsha) [inline README.md]: Add section Redis to the config example

File: README.md [hunk 20]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| redis.clusterServersConfig.nodeAddresses   | -                  | No|Json array with Redis cluster server addresses, e.g. ["redis://host1:port1","redis://host2:port2"]
+| invitations.ttlInSeconds                   | 259200             | No|Invitation time to live in seconds
+| access.admin.rules                         | -                  | No|Matches claims from identity providers with the rules to figure out whether a user is allowed to perform admin actions, like deleting any resource or approving a publication. Example: [{"source": "roles", "function": "EQUAL", "targets": ["admin"]}]. If roles contain "admin, the actions are allowed.
 
 #### Google IdP
 
 The setting `identityProviders.*.rolePath` should be set `fn:getGoogleWorkspaceGroups`.
```
Comment (sr-remsha) [inline README.md]: consider adding it in the description of identity provider parameters for a better usability

File: README.md [hunk 21]
```diff
@@ -33,49 +33,51 @@ Static settings are used on startup and cannot be changed while application is r
...
 
+<!-- Do we want to add AWS storge example?-->
```
Comment (sr-remsha) [inline README.md]: Please provide an example for AWS

File: README.md (hunk 19 above)
Comment (sr-remsha) [reply README.md]: Is it a required section starting from 0.8?

File: README.md [hunk 22]
```diff
@@ -25,6 +25,8 @@ Or run com.epam.aidial.core.AIDial class from your favorite IDE.
 
 ## Configuration
 
+<!--Mention that there is a Dynamic configuration an well and give a link -->
```
Comment (sr-remsha) [inline README.md]: Please add a link to the corresponding section and explain what kind of config to use and for which purpose

File: README.md [hunk 23]
```diff
@@ -159,6 +163,8 @@ Note: Redis will be strictly required in the upcoming releases 0.8+.
 
 ### Dynamic settings
 
+<!--Provide a link to an example-->
```
Comment (sr-remsha) [inline README.md]: Add a link to an example

File: README.md [hunk 24]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
+| Parameter                             | Description      |
+|---------------------------------------|------------------------------------------------------------------------------------------|
+| routes | Path(s) for specific upstream routing or to respond with a configured body.   |
```
Comment (sr-remsha) [inline README.md]: Add example to the sample file

Comment (sr-remsha) [reply README.md]: We do not have here Addons section

File: README.md [hunk 25]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
+| applications                          | A list of deployed AI DIAL Applications and their parameters:<br />`<application_name>`: Unique application name.                           |
+| applications.<application_name>       | `endpoint`: AI DIAL Application API for chat completions.<br />`iconUrl`: Icon path for the AI DIAL Application on UI.<br />`description`: Brief AI DIAL Application description.<br />`displayName`: AI DIAL Application name on UI.<br />`inputAttachmentTypes`: A list of allowed MIME types for the input attachments.<br />`maxInputAttachments`: Maximum number of input attachments (default is zero when `inputAttachmentTypes` is unset, otherwise, infinity) <br/> `forwardAuthToken`: If flag is set to `true` forward Http header with authorization token to chat completion endpoint of the application.       |
```
Comment (sr-remsha) [inline README.md]: Add a full example to the sample file

File: README.md (hunk 24 above)
Comment (sr-remsha) [reply README.md]: We do not have Assistant section here

File: README.md [hunk 26]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
+| models | A list of deployed models and their parameters:<br />`<model_name>`: Unique model name.                      |
 | models.<model_name>                   | `type`: Model type—`chat` or `embedding`.<br />`iconUrl`: Icon path for the model on UI.<br />`description`: Brief model description.<br />`displayName`: Model name on UI.<br />`displayVersion`: Model version on UI.<br />`endpoint`: Model API for chat completions or embeddings.<br />`tokenizerModel`: Identifies the specific model whose tokenization algorithm exactly matches that of the referenced model. This is typically the name of the earliest-released model in a series of models sharing an identical tokenization algorithm (e.g. `gpt-3.5-turbo-0301`, `gpt-4-0314`, or `gpt-4-1106-vision-preview`). This parameter is essential for DIAL clients that reimplement tokenization algorithms on their side, instead of utilizing the `tokenizeEndpoint` provided by the model.<br />`features`: Model features.<br />`limits`: Model token limits.<br />`pricing`: Model pricing.<br />`upstreams`: Used for load-balancing—request is sent to model endpoint containing X-UPSTREAM-ENDPOINT and X-UPSTREAM-KEY headers. |
```
Comment (sr-remsha) [inline README.md]: Add one full example to the sample file

File: README.md [hunk 27]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
-| models.<model_name>.limits            | `maxPromptTokens`: maximum number of tokens in a completion request.<br />`maxCompletionTokens`: maximum number of tokens in a completion response.<br />`maxTotalTokens`: maximum number of tokens in completion request and response combined.<br />Typically either `maxTotalTokens` is specified or `maxPromptTokens` and `maxCompletionTokens`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                            |
```
Comment (sr-remsha) [inline README.md:180]: Add example to the sample config file

File: README.md [hunk 28]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
-| models.<model_name>.pricing           | `unit`: the pricing units (currently `token` and `char_without_whitespace` are supported).<br />`prompt`: per-unit price for the completion request in USD.<br />`completion`: per-unit price for the completion response in USD.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```
Comment (sr-remsha) [inline README.md:181]: Add example to the sample config file

File: README.md [hunk 29]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
-| models.<model_name>.features          | `rateEndpoint`: endpoint for rate requests *(exposed by core as `<deployment name>/rate`)*.<br />`tokenizeEndpoint`: endpoint for requests to the model tokenizer *(exposed by core as `<deployment name>/tokenize`)*.<br />`truncatePromptEndpoint`: endpoint for truncating prompt requests *(exposed by core as `<deployment name>/truncate_prompt`)*.<br />`systemPromptSupported`: does the model support system prompt (default is `true`).<br />`toolsSupported`: does the model support tools (default is `false`).<br />`seedSupported`: does the model support `seed` request parameter (default is `false`).<br />`urlAttachmentsSupported`: does the model/application support attachments with URLs (default is `false`).<br />`folderAttachmentsSupported`: does the model/application support folder attachments (default is `false`)                                                                                                                                                                                            |
```
Comment (sr-remsha) [inline README.md:182]: Add example to the sample config file

File: README.md [hunk 30]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
+| models.<model_name>.upstreams         | `endpoint`: Model endpoint.<br />`key`: Your API key.                         |
+| keys   | API Keys parameters:<br />`<core_key>`: Your API key.                         |
+| keys.<core_key>                       | `project`: Project name assigned to this key.<br />`role`: A configured role name that defines key permissions.                             |
+| roles  | API key roles `<role_name>` with associated limits. Each API key has one role defined in the list of roles. Roles are associated with models, applications, assistants, and defined limits.               |
+| roles.<role_name>                     | `limits`: Limits for models, applications, or assistants.                     |
+| roles.<role_name>.limits              | `minute`: Total tokens per minute limit sent to the model, managed via floating window approach for well-distributed rate limiting.<br />`day`: Total tokens per day limit sent to the model, managed via floating window approach for balanced rate limiting.          |
```
Comment (sr-remsha) [inline README.md]: Describe other parameters mentioned in the sample file such as search, forecast, calculator, search assistant, app

File: README.md [hunk 31]
```diff
@@ -170,22 +176,22 @@ Dynamic settings include:
...
 
 ## License
```
Comment (sr-remsha) [inline README.md:210]: 2023 --> 2024

File: README.md [hunk 32]
```diff
@@ -159,6 +163,8 @@ Note: Redis will be strictly required in the upcoming releases 0.8+.
 
 ### Dynamic settings
 
+<!--Provide a link to an example-->
+
 Dynamic settings are stored in JSON files, specified via "config.files" static setting, and reloaded at interval,
 specified via "config.reload" static setting.
 Dynamic settings include:
```
Comment (sr-remsha) [inline README.md]: Make the list consistent with the actual section in the config

File: README.md (hunk 3 above)
Comment (sr-remsha) [reply README.md]: Add link to the Dynamic config section

File: README.md [hunk 33]
```diff
@@ -33,49 +35,51 @@ Static settings are used on startup and cannot be changed while application is r
...
+| redis.clusterServersConfig.nodeAddresses   | -                  | No|Json array with Redis cluster server addresses, e.g. ["redis://host1:port1","redis://host2:port2"]
+| invitations.ttlInSeconds                   | 259200             | No|Invitation time to live in seconds
+| access.admin.rules                         | -                  | No|Matches claims from identity providers with the rules to figure out whether a user is allowed to perform admin actions, like deleting any resource or approving a publication. Example: [{"source": "roles", "function": "EQUAL", "targets": ["admin"]}]. If roles contain "admin, the actions are allowed.
 
 #### Google IdP
 
 The setting `identityProviders.*.rolePath` should be set `fn:getGoogleWorkspaceGroups`.
 
+<!-- Do we want to add AWS storge example?-->
+
```
Comment (sr-remsha) [inline README.md:99]: Consider adding an additional section dedicated to persistent storage parameters and description

File: README.md (hunk 22 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 4 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 5 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 6 above)
Comment (astsiapanay) [reply README.md]: There are multiple references. Per IDP. Need to wait for https://github.com/epam/ai-dial/tree/idp_configuration/docs/Deployment/idp-configuration completion

File: README.md (hunk 7 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 8 above)
Comment (astsiapanay) [reply README.md]: Added

File: README.md (hunk 9 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 11 above)
Comment (astsiapanay) [reply README.md]: added to example

File: README.md (hunk 10 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 13 above)
Comment (astsiapanay) [reply README.md]: added default value

File: README.md (hunk 14 above)
Comment (astsiapanay) [reply README.md]: Some blob storages require an endpoint to be provided. So the setting is optional.

File: README.md (hunk 15 above)
Comment (astsiapanay) [reply README.md]: The examples are in the doc below.

File: README.md (hunk 16 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 17 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 18 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 19 above)
Comment (astsiapanay) [reply README.md]: yes, it's required fixed.

File: README.md (hunk 21 above)
Comment (astsiapanay) [reply README.md]: I suppose it's already described. I don;t know what to add.

File: README.md (hunk 33 above)
Comment (astsiapanay) [reply README.md:99]: I suppose it's already described. I don;t know what to add.

File: README.md (hunk 23 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 25 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 26 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 27 above)
Comment (astsiapanay) [reply README.md:180]: added

File: README.md (hunk 28 above)
Comment (astsiapanay) [reply README.md:181]: added

File: README.md (hunk 29 above)
Comment (astsiapanay) [reply README.md:182]: added

File: README.md (hunk 31 above)
Comment (astsiapanay) [reply README.md:210]: fixed

File: README.md (hunk 4 above)
Comment (sr-remsha) [reply README.md]: What about Okta? @astsiapanay

File: README.md (hunk 1 above)
Comment (astsiapanay) [reply README.md]: that makes sense to me.

File: README.md (hunk 12 above)
Comment (astsiapanay) [reply README.md]: @astsiapanay  need to provide a reference to vertex settings

File: README.md (hunk 13 above)
Comment (astsiapanay) [reply README.md]: @astsiapanay to add a note: depends on a specific provider.

Comment (astsiapanay) [reply README.md]: @astsiapanay to add a section for AWS s3

File: README.md (hunk 24 above)
Comment (astsiapanay) [reply README.md]: @astsiapanay  to check wordings in new descriptions

File: README.md (hunk 30 above)
Comment (astsiapanay) [reply README.md]: @astsiapanay to add default limits, review role and limit descriptions

File: README.md (hunk 33 above)
Comment (astsiapanay) [reply README.md:99]: @astsiapanay add summary descriptions about storage types: blob and key-value. why, what

File: README.md (hunk 2 above)
Comment (astsiapanay) [reply README.md]: @astsiapanay add a lint to dynamic config example. change description because this one confuses.

Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (astsiapanay) [reply README.md]: added

File: README.md (hunk 12 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 13 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 16 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 24 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 30 above)
Comment (astsiapanay) [reply README.md]: fixed

File: README.md (hunk 33 above)
Comment (astsiapanay) [reply README.md:99]: fixed

**Review by astsiapanay — COMMENTED** [review]
done

**Review by astsiapanay — APPROVED** [review]

**Pull request merged**